src/NUnitFramework/framework/Constraints/EqualityAdapter.cs

```diff
@@ -52,6 +52,12 @@ public virtual bool CanCompare(object x, object y)
             if (x is string && y is string)
                 return true;
 
+            if (x is string && !(y is IEnumerable) ||
+                y is string && !(x is IEnumerable))
+            {
+                return true;
+            }
+
             if (x is IEnumerable || y is IEnumerable)
                 return false;
```

Comment:
If desired, the whole method impl could be:

```cs
return (x is string || !(x is IEnumerable))
    && (y is string || !(y is IEnumerable));
```

Reply:
@jnm2 I like it, added a commit to the branch doing so.